assets/src/domain/eventEditor/ui/ticketAssignmentsManager/data/useAssignmentManager.ts

```diff
@@ -77,18 +77,24 @@ const useAssignmentManager = (): AM => {
 		}
 	};
 
-	const initialize: AM['initialize'] = (data: TAMRelationalData, forDate) => {
+	const initialize: AM['initialize'] = ({ data, assignmentType, entityId }) => {
 		const relationsToPick: Array<keyof TAMRelationalData> = ['datetimes', 'tickets'];
```

Comment:
:+1: 
we might want to have a cheatsheet at some point of small types like these